lighty-core/dependency-versions/pom.xml

```diff
@@ -164,6 +164,11 @@
                 <artifactId>javax.ws.rs-api</artifactId>
                 <version>2.1.1</version>
             </dependency>
+            <dependency>
+                <groupId>javax.activation</groupId>
+                <artifactId>activation</artifactId>
+                <version>1.1.1</version>
```

Comment:
Is this really necessary? AFAICT this declaration already comes from odlparent

Reply:
I don't know about that we use odlparent at lighty.core. And also in dependence tree I don't see any other declaration of javax.activation dependence.

Reply:
Ok, I see your point. Fixed. I didn't noticed that this dependence has already version management. Thank you.